dvc/stage.py

```diff
@@ -663,7 +663,7 @@ def dump(self):
 
         self._check_dvc_filename(fname)
 
-        logger.info(
+        logger.debug(
             "Saving information to '{file}'.".format(file=relpath(fname))
```

Comment:
And one last question: Is this really a debug output? I also find that message useful in the output.

Reply:
the thing is the tracked file name will be printed in the progress bar itself, while the dvc file names will all be printed at the end. This means this message should no longer be required. I left it in for 1. debugging and 2. not sure if the 12 other uses of `stage.dump()` actually need it to be `logger.info`. This is the only real issue I have with this PR atm.

Reply:
Cool. Feel fee to ask @efiop and/or resolve this conversation when you see fit. Thanks.